src/queries/catalog/call-context-query/identify-link-to-last-call-relation.ts

```diff
@@ -121,7 +121,7 @@ export function identifyLinkToLastCallRelation(
 	graph: DataflowGraph,
 	{ callName, ignoreIf, cascadeIf }: LinkTo<RegExp>
 ): NodeId[] {
-	if(ignoreIf && ignoreIf(from, graph)) {
+	if(ignoreIf !== undefined && (typeof ignoreIf === 'function' ? ignoreIf(from, graph) : ignoreIf)) {
```

Comment:
No. Please add a constant `() => true` in that case and do not use a constant. This would break other things.